caravel/models.py

```diff
@@ -1101,6 +1101,13 @@ def fetch_metadata(self):
                     metric_type='sum',
                     expression="SUM({})".format(quoted)
                 ))
+            if dbcol.sum:
```

Comment:
dbcol.max and dbcol.min are also in similar relation.
@mistercrunch  - should avg be a separate field in the TableColumn ?